RUFAS/biophysical/animal/animal_growth/animal_growth.py

```diff
@@ -1,2 +1,412 @@
+            updated_conceptus_weight = 0
+
+        elif general_properties.days_in_preg > 50:
+            conceptus_total_weight = (
+                0.0148 * reproduction_properties.gestation_length - 2.408
+            ) * reproduction_properties.calf_birth_weight
+            conceptus_param = conceptus_total_weight ** (1 / 3) / (reproduction_properties.gestation_length - 50)
+            conceptus_growth = 3 * conceptus_param**3 * (general_properties.days_in_preg - 50) ** 2
+            updated_conceptus_weight += conceptus_growth
+        else:
+            conceptus_growth = 0
+        return conceptus_growth, updated_conceptus_weight
+
+    @staticmethod
+    def _calculate_cow_conceptus_growth(
+        animal_growth_properties: AnimalGrowthProperties,
+        reproduction_properties: ReproductionProperties,
+        general_properties: GeneralProperties,
+    ) -> tuple[float, float, float]:
+        """
+        Calculates the conceptus growth for cows.
+
+        Parameters
+        ----------
+        animal_growth_properties: AnimalGrowthProperties
+            Animal properties that are related to body weight growth.
+        reproduction_properties: ReproductionProperties
+            Animal properties that are related to animal reproduction.
+        general_properties: GeneralProperties
+            Animal properties that are general or are used to determine many animal outcomes.
+
+        Returns
+        -------
+        tuple[float, float, float]
+            The conceptus growth for pregnant heifers (kg), the updated conceptus weight (kg), and the updated
+            tissue changed (kg).
+        """
+        updated_tissue_change = (
+            0
+            if general_properties.days_in_preg == reproduction_properties.gestation_length
+            else animal_growth_properties.tissue_changed
+        )
+
+        conceptus_growth, updated_conceptus_weight = AnimalGrowth._calculate_pregnant_heifer_conceptus_growth(
+            reproduction_properties, general_properties
+        )
+
+        return conceptus_growth, updated_conceptus_weight, updated_tissue_change
+
+    @staticmethod
+    def _calculate_pregnant_heifer_target_daily_growth(
+        reproduction_properties: ReproductionProperties, general_properties: GeneralProperties
+    ) -> float:
+        """
+        Calculates the target daily growth for pregnant heifers.
+
+        Parameters
+        ----------
+        reproduction_properties: ReproductionProperties
+            Animal properties that are related to animal reproduction.
+        general_properties: GeneralProperties
+            Animal properties that are general or are used to determine many animal outcomes.
+
+        Returns
+        -------
+        float
+            The daily growth rate for pregnant heifers (kg).
+        """
+        divisor = reproduction_properties.gestation_length - general_properties.days_in_preg
+        if divisor == 0:
+            divisor = 1
```

Comment:
Should this also use `abs()`?
If so:

```suggestion
        divisor = max(1, abs(reproduction_properties.gestation_length - general_properties.days_in_preg))
```